data/common.yaml

```diff
@@ -0,0 +1,10 @@
+---
+systemd::service_limits: {}
+systemd::manage_resolved: false
+systemd::resolved_ensure: 'running'
+systemd::manage_networkd: false
+systemd::networkd_ensure: 'running'
+systemd::manage_timesyncd: false
+systemd::timesyncd_ensure: 'running'
+systemd::ntp_server: ~
```

Comment:
I'm not sure what the best practice is, but I think it's cleaner to have any ```Optional``` type in ```init.pp``` default to ```undef```.

Reply:
AFAIK best practice is ~, which is also used in the puppetlabs modules.

Reply:
I guess it would be more consistent to have all parameters in hiera.